### ISSUE TITLE
BASIC programmation

### ISSUE THREAD

**Issue by windows9x95** [Omega-Numworks/Omega#254]
#### What I want to see in the next version of Omega
Hi, I know the basic programmation is an interpreted language so it don't need to be compilated. So I want to program with it in my Numworks.
I really want it, for prove my skills in BASIC :)
Thanks you.

**Comment by 0b101** [comment]
You might be able to make your own simple BASIC interpreter in Python on your calculator

**Comment by windows9x95** [comment]
I think that can be good to do it in a native application, so this is for that I ask.

**Comment by 0b101** [comment]
Like TinyBASIC or something like that?

**Comment by windows9x95** [comment]
Yes that can be cool, but this is hard or not to include it (if possible) ?
